defaults/main.yml

```
@@ -22,7 +22,7 @@ assessment_tool_go_build: true
 assessment_tool_group: root
 
 # Mode for the install directory
-assessment_tool_mode: 0775
+assessment_tool_mode: u=rwx,g=rx,o=rx
```

Just when I was beginning to doubt if I would ever spot one of these conversions that was incorrect:
```suggestion
assessment_tool_mode: u=rwx,g=rwx,o=rx
```

Then again, maybe you made this permissions change on purpose?  Let us know either way.

---

Thank you for catching that!